src/test/kotlin/org/stellar/walletsdk/helpers/objectFromJsonFile.kt

```diff
@@ -0,0 +1,15 @@
+package org.stellar.walletsdk.helpers
+
+import com.google.gson.Gson
+import java.io.File
+
+fun <T> objectFromJsonFile(filePath: String, t: Class<T>): T {
+  val jsonString =
+    try {
+      File(filePath).readText()
+    } catch (e: Exception) {
```

Comment:
There are many exceptions that may happen while reading a file. Throwing `Error("File not found")` can be mis-leading later.

You may leave the exception un-caught and let the caller handle the Exception.